back-end/test/getUser.js

```diff
@@ -1,43 +1,48 @@
+/*
 const chai = require('chai')
 const chaiHttp = require('chai-http')
 const axios = require('axios')
 const sinon = require('sinon')
 const expect = chai.expect
 const server = require('../app.js')
+const User=require('../models/User')
 const { mockErrorMessage, mockUserAPI } = require('../MockData.js')
 chai.use(chaiHttp)
 
-describe('GET /tv_users/1', () => {
+
+describe('GET /tv_users/77', () => {
   let stub
 
   afterEach(() => {
     stub.restore()
   })
 
+//sinon.stub(modelName, 'findOne').returns({object to be returned})
   it('should return 200 OK and data for a valid id', async () => {
-    stub = sinon.stub(axios, 'get').resolves({
+    stub = sinon.stub(User, 'findOne').returns({
```

Comment:
Were you able to get these tests working? Or are they commented out because there were issues?

Reply:
i probably will create another task for testing hehe; for now will revert to how it was probably